py/selenium/webdriver/chromium/options.py

```diff
@@ -126,6 +127,39 @@ def add_experimental_option(self, name: str, value: Union[str, int, dict, list[s
         """
         self._experimental_options[name] = value
 
+    @property
+    def enable_webextensions(self) -> bool:
+        """Returns whether webextension support is enabled for Chromium-based browsers.
+
+        :Returns: True if webextension support is enabled, False otherwise.
+        """
+        return self._enable_webextensions
+
+    @enable_webextensions.setter
+    def enable_webextensions(self, value: bool) -> None:
+        """Enables or disables webextension support for Chromium-based browsers.
+
+        When enabled, this automatically adds the required Chromium flags:
+        - --enable-unsafe-extension-debugging
+        - --remote-debugging-pipe
+
+        :Args:
+         - value: True to enable webextension support, False to disable.
+        """
+        self._enable_webextensions = value
+        if value:
+            # Add required flags for Chromium webextension support
+            required_flags = ["--enable-unsafe-extension-debugging", "--remote-debugging-pipe"]
+            for flag in required_flags:
+                if flag not in self._arguments:
+                    self.add_argument(flag)
+        else:
+            # Remove webextension flags if disabling
+            flags_to_remove = ["--enable-unsafe-extension-debugging", "--remote-debugging-pipe"]
```

Comment:
Just wanted to note that from my reading of the `--remote-debugging-pipe` [description](https://peter.sh/experiments/chromium-command-line-switches/#remote-debugging-pipe) it is not exclusively a webextension command line switch. So if one adds it separately and chooses to disable web extensions via this method, as in,

```python
    options = webdriver.ChromeOptions()
    options.add_argument("--remote-debugging-pipe")
    options.enable_webextensions = False
```

 it will negate that command line switch and might be tough to debug. Probably the method documentation should note that disabling removes both switches.

Reply:
This was discussed but no conclusion was made - https://github.com/SeleniumHQ/selenium/pull/15794#discussion_r2113162844, what do you think would be the best way to resolve this? 

Adding docs is good.